src/utils/pipelineStatusValues.js

```diff
@@ -8,7 +8,7 @@ export default {
   TIMED_OUT: 'TIMED_OUT',
   ABORTED: 'ABORTED',
   SUCCEEDED: 'SUCCEEDED',
-
+  NEEDS_RERUN: 'NEEDS_RERUN',
```

Comment:
We won't need this if we remove the checks for gem2s in the ui, right?

Reply:
Maybe i can remove the checks and pass the status to the api? So the checks are done there?